test/runtime.js

```diff
@@ -19,7 +26,75 @@
 /* TODO: images */
 /* TODO: math */
 /* TODO: random */
-/* TODO: vectors */
+
+/* vectors */
+
+var iota = 0.0001;
+function equalish(a,b){
```

Comment:
I already defined `assert.fuzzyEqual` in [test-helpers.js](https://github.com/waterbearlang/waterbear/blob/master/test/test-helpers.js#L18).